### ISSUE TITLE
Crash in End with physics mod ocean enabled

### ISSUE THREAD

**Issue by Syclusion** [X0nk/Bliss-Shader#224]
### Describe the bug
Using 1.20.1 forge with newest embeddium and oculus + dh, going to the end with the physics mod and rendering water causes crash.

2. If you cannot do option 1. Please list the following information instead:
- Nvidia
- Windows
-1.20.1
- Oculus
- Newest version of bliss from github as of the 26th
-DH + physics

### Describe how to reproduce the bug
Go to end with physics mod ocean, see water = crash

[crash-2024-06-26_16.55.23-client.txt](https://github.com/user-attachments/files/15996710/crash-2024-06-26_16.55.23-client.txt)

**Comment by X0nk** [comment]
bliss doesnt support the physics mod water. have you tested to make sure its shader related? does it happen with other shaders? does it happen *without* shaders on?

**Comment by Syclusion** [comment]
Yup, originally posted this in the physics mod github. Traced it to bliss, without shaders no crash. With bsl no crash.

**Comment by X0nk** [comment]
does it happen with physics mod ocean toggled off?

if that doesn't stop it, im not sure i can do anything about it. bliss isn't even marked as having physics mod support using physics mod's provided things. there is *zero* code for anything related to physics mod at all.

**Comment by Syclusion** [comment]
Yes when toggled off the crash stops

**Comment by Endri2015** [comment]
well it works for me and are you in the latest version of these mods?

**Comment by Syclusion** [comment]
> well it works for me and are you in the latest version of these mods?

Yes, still broken. Not sure how its working for you

**Comment by Syclusion** [comment]
Just did a sanity check as well, only mods loaded were newest versions of physics mod embeddium distant horizons and oculus. Insta crash when following the same steps

**Comment by Syclusion** [comment]
> does it happen with physics mod ocean toggled off?
> 
> if that doesn't stop it, im not sure i can do anything about it. bliss isn't even marked as having physics mod support using physics mod's provided things. there is _zero_ code for anything related to physics mod at all.

Sorry for the quote but i figured out the error, it comes from the iris compat toggle for versions under 1.7.0
